### PR TITLE
added skip and retry option to a location state

### PR DESCRIPTION
**Pull request by g1039**
@hodgestar Ready for review

### CODE REVIEW

File: go-app-ussd.js [hunk 1]
```diff
@@ -1238,10 +1238,14 @@ go.app = function() {
...
-                next_text: 'More',
-                previous_text: 'Back',
+                next_text: $('More'),
+                previous_text: $('Back'),
+                retry_text: $('Retry'),
+                skip_text: $('Skip'),
+                
 
                 map_provider: new OpenStreetMap({
+                    api_key: 'XXX',
```
Comment (hodgestar) [inline go-app-ussd.js:1248]: We need to create a ticket for fixing this properly (location searches won't work until it is fixed).

File: package.json [hunk 2]
```diff
@@ -22,12 +22,12 @@
   },
   "homepage": "https://github.com/praekelt/go-refugeerights",
   "dependencies": {
-    "vumigo_v02": "~0.2",
-    "q": "~0.9",
+    "go-jsbox-location": "^0.2.2",
```
Comment (hodgestar) [inline package.json]: We should change the version to `~0.2.3`.

**Comment by hodgestar** [comment]
Many of the fixture changes (and resulting test changes) are unnecessary (only the poifinder API request needed its lat and lon updated).

**Comment by hodgestar** [comment]
Shout when it's ready for re-review.

**Comment by hodgestar** [comment]
After some examination and discussion with @TrevorOctober we decided that it is actually best to update all the fixtures since we have to change which option is selected (since option 3 is no longer displayed in most cases).

**Comment by g1039** [comment]
@hodgestar Ready for review

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**